src/redux/reducers.js

```diff
@@ -1265,6 +1265,14 @@ export function sendTo(state = INITIAL_SEND_TO_STATE, action) {
         feeds: state.feeds
       }
     }
+    case response(ActionTypes.CREATE_GROUP): {
+      let groupId = action.payload.groups.id
+      let group = userParser(action.payload.groups)
+      return {
+        expanded: state.expanded,
```

Comment:
This line should read as `...state`, to be in the same vein as other state returns in our code. :)